src/bundle/Resources/public/js/scripts/admin.location.changeLanguage.js

```diff
@@ -0,0 +1,7 @@
+(function (global, doc) {
+    const changeLocationLanguage = event => {
+        window.location = event.target.options[event.target.selectedIndex].value;
```

Comment:
I think select should have it in `value`, so `event.currentTarget.value` should be enough.